mlir/test/Conversion/ArithToEmitC/arith-to-emitc.mlir

```diff
@@ -177,3 +177,42 @@ func.func @arith_int_to_float_cast_ops(%arg0: i8, %arg1: i64) {
 
   return
 }
+
```

Comment:
If seems the other test names in the file start with `aith_`, can you align the new tests to that (also for the unsupported ones)?

Reply:
Done!